src/lib/registration/registration_codes_loader.rb

```diff
@@ -46,9 +44,10 @@ def reg_codes_from_usb_stick
       with_tempfile("regcodes-") do |path|
         REGCODES_NAME_HANDLERS.each do |name, handler|
           next unless get_file_from_url(scheme: "usb", host: "",
-                                        urlpath: "/#{name}",
-                                        localfile: path,
-                                        urltok: {}, destdir: "")
+            urlpath: "/#{name}",
+            localfile: path,
+            urltok: {}, destdir: "")
```

Comment:
This example of an inconsistency, for multiline string it would require the extra indentation here.